nym-node-status-api/src/testruns/mod.rs

```diff
@@ -74,3 +76,11 @@ async fn run(pool: &DbPool) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[instrument(level = "debug", skip_all)]
+async fn refresh_stale_testruns(pool: &DbPool, refresh_interval: Duration) -> anyhow::Result<()> {
+    let chrono_duration = chrono::Duration::from_std(refresh_interval)?;
```

Comment:
we're using `time` almost everywhere in the codebase rather than `chrono`, if it's an easy change, would it be possible if you migrated to that one instead? if not, then ignore this comment

Reply:
`chrono` was already being used but I'll give it a shot.